guides/source/getting_started.md

```diff
@@ -609,7 +608,7 @@ end
 
 If you re-submit the form now, you may not see any change on the page. Don't worry!
 This is because Rails by default returns `204 No Content` response for an action if
-we don't specify what the response should be. We just added the `create` action
+we don't specify what the response should be. We added the `create` action
```

Comment:
Think "just" here is used as "what just happened" to connect to the past sentence. Don't think it's right to remove, but happy to hear the ification (you'll have to add back a `just` there 😂).

Reply:
Ah interesting - I can see that read of it. Looking through it, my interpretation is:
1) We tell the reader to define a `create` action to make the "Unknown action" go away
2) We warn them that they are unlikely to see changes on the page (because we added the action, but didn't specify how the response should be (ie - no logic in the action yet).

I don't think the "just" here is used in a "simply" sense, but I felt it was unneeded, and potentially confusing. I'm happy to put it back in though - let me know your thoughts!